xbmc/application/Application.cpp

```diff
@@ -2253,43 +2254,10 @@ bool CApplication::PlayMedia(CFileItem& item, const std::string& player, PLAYLIS
   return PlayFile(item, player, false);
 }
 
-// PlayStack()
-// For playing a multi-file video.  Particularly inefficient
-// on startup, as we are required to calculate the length
-// of each video, so we open + close each one in turn.
-// A faster calculation of video time would improve this
-// substantially.
-// return value: same with PlayFile()
-bool CApplication::PlayStack(CFileItem& item, bool bRestart)
-{
-  const auto stackHelper = GetComponent<CApplicationStackHelper>();
-  if (!stackHelper->InitializeStack(item))
-    return false;
-
-  std::optional<int64_t> startoffset = stackHelper->InitializeStackStartPartAndOffset(item);
-  if (!startoffset)
-  {
-    CLog::LogF(LOGERROR, "Failed to obtain start offset for stack {}. Aborting playback.",
-               item.GetDynPath());
-    return false;
-  }
-
-  CFileItem selectedStackPart = stackHelper->GetCurrentStackPartFileItem();
-  selectedStackPart.SetStartOffset(startoffset.value());
-
-  if (item.HasProperty("savedplayerstate"))
-  {
-    selectedStackPart.SetProperty("savedplayerstate", item.GetProperty("savedplayerstate")); // pass on to part
-    item.ClearProperty("savedplayerstate");
-  }
-
-  return PlayFile(selectedStackPart, "", true);
-}
-
-bool CApplication::PlayFile(CFileItem item,
-                            const std::string& player,
-                            bool bRestart /* = false */,
-                            bool forceSelection /* = false */)
+bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRestart)bool CApplication::PlayFile(CFileItem item,
```

Comment:
```suggestion
bool CApplication::PlayFile(CFileItem item,
```

Reply:
Thanks. Spotted this and fixed locally. Just adding bluray:// support to folder stacks (so can navigate between items like with files) and will then update all together.